script.js

```diff
@@ -17,40 +17,40 @@ const cardContainer = [
     sourceLink: 'https://github.com/yonas44/space-tourism-website',
   },
   {
-    image: 'Icons/picture/space-tourism.jpg',
-    title: 'Second Project',
+    image: './Icons/picture/modal_picture.png',
+    title: 'Keeping track of hundreds of components',
```

Comment:
- [ ] As a previous reviewer state, kindly make sure when click on see project button the corresponding project displayed in the modal instead of using in the same data in the array. It will be handled dynamically with JavaScript.

Reply:
@omarsalem7 thank you for your feedback but I used the same title and image because I only had one actual project on my github, so shall I add images and other contents from another template cards?